mobile_back_qti/cpp/backend_qti/qti_settings_sd7g1.h

```diff
@@ -0,0 +1,144 @@
+/* Copyright (c) 2023 Qualcomm Innovation Center, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef QTI_SETTINGS_SD7G1_H
+#define QTI_SETTINGS_SD7G1_H
+
+const std::string qti_settings_sd7g1 = R"SETTINGS(
+common_setting {
+  id: "num_threads"
+  name: "Number of threads"
+  value {
+    value: "4"
+    name: "4 threads"
+  }
+}
+
+benchmark_setting {
+  benchmark_id: "image_classification"
+  accelerator: "snpe_dsp"
+  accelerator_desc: "HTP"
+  framework: "SNPE"
+  custom_setting {
+    id: "bg_load"
+    value: "true"
+  }
+  model_path: "local:///mlperf_models/mobilenet_edgetpu_224_1.0_htp.dlc"
```

Comment:
@mohitmundhragithub The models are published so this one should be replaced with `https` link, right?

Reply:
updated in the latest commit.